### PR TITLE
android: fixed connection failed with status code 133

### LINKED ISSUE #646: Connection Failed on Android: Stauts 133
Hi,

I noticed that whenever I try to connect to my bluetooth device on Android, the connection fails and I get the following error:

```
D/BluetoothGatt: onClientConnectionState() - status=133 clientIf=9 device=00:13:43:27:2D:10
```

I tested it on the following devices:
* Google Pixel 3a running Android 10
* Samsung Galaxy Tab S5e (SM-T720) running Android 10

### PR DESCRIPTION
**Pull request by p-andreas**
fixes #646

### CODE REVIEW

File: src/android/BluetoothLePlugin.java [hunk 1]
```diff
@@ -1480,7 +1480,12 @@ private void connectAction(JSONArray args, CallbackContext callbackContext) {
 
     connections.put(device.getAddress(), connection);
 
-    BluetoothGatt bluetoothGatt = device.connectGatt(cordova.getActivity().getApplicationContext(), autoConnect, bluetoothGattCallback);
+    BluetoothGatt bluetoothGatt = null;
+    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
+      bluetoothGatt = device.connectGatt(cordova.getActivity().getApplicationContext(), autoConnect, bluetoothGattCallback, BluetoothDevice.TRANSPORT_LE);
```
Comment (randdusing) [inline src/android/BluetoothLePlugin.java:1485]: I'm slightly concerned with this breaking behavior for people who relied on TRANSPORT_AUTO. Could you make this a parameter instead? If version >= M and a transport has been specified, call #connectGatt() with a transport type.

Comment (p-andreas) [reply src/android/BluetoothLePlugin.java]: Please check out the updated version

**Comment by randdusing** [comment]
Thanks, push 6.1.0

**Comment by EinfachHans** [comment]
just additional info: These changes are currently not included in `readme`

**Comment by randdusing** [comment]
@EinfachHans Thanks, I updated the readme

**Pull request merged**